.github/workflows/validation-jobs.yml

```diff
@@ -49,16 +48,6 @@ jobs:
 
       - uses: dtolnay/rust-toolchain@stable
 
-      - uses: actions/cache@v4
-        with:
-          path: |
-            ~/.cargo/bin/
-            ~/.cargo/registry/index/
-            ~/.cargo/registry/cache/
-            ~/.cargo/git/db/
-            target/
-          key: ${{ runner.os }}-cargo-build-android-${{ hashFiles('**/Cargo.toml') }}
-
```

Comment:
Aren't these all parallel jobs and thus each of them will still need to retrieve cache? Or is there something I'm missing here? In other words, I'm questioning these removals as they aren't being replaced with the `cargo-cache` action but just being removed.

Reply:
Though it has been a while, I believe my reasoning was two-parted:

1. We have a cache storage limit of 10 GB, and Bevy builds take between 500 MB and 3 GB. We don't need validation checks to be fast, since they ideally only happen once, so its more valuable to make space for the fast-iteration checks run on every PR update.
2. Caches _can_ add a layer of instability. Since validation checks are the "final shield" between a PR and being merged, we probably don't want to risk it.

Reply:
Gotcha! So with that clarified, is there a reason why `build-and-install-on-iOS` still uses cache while the rest of the validation jobs won't anymore? Sorry if these were previously clarified in this thread, but I can't find any context on this. Does this have to do with what you said here?

> Building on MacOS and docs are the two longest jobs. I'm going to see if I can shorten their duration, if possible.

Thanks for answering and for your patience!

Reply:
> is there a reason why `build-and-install-on-iOS` still uses cache while the rest of the validation jobs won't anymore?

I think that was a mistake, let me remove it!